macstadium-orka-server/src/main/java/com/macstadium/orka/client/OrkaClient.java

```diff
@@ -171,6 +172,15 @@ private String executeCall(Request request) throws IOException {
 
     @Override
     public void close() throws IOException {
-        this.delete(this.endpoint + TOKEN_PATH, "");
+        String response = this.get(this.endpoint + HEALTH_PATH);
+        Gson gson = new Gson();
+        JsonObject jsonObject = gson.fromJson(response, JsonObject.class);
+        String apiVersionString = jsonObject.get("api_version").toString();
+        apiVersionString.replace(".", "");
+        int apiVersion = Integer.parseInt(apiVersionString);
+
+        if (apiVersion < 210) {
```

Comment:
The first version we use a single token is 2.1.1.
So shouldn't this be `< 211`?

Reply:
Ah. I just saw that we do this [here](https://github.com/macstadium/orka-integrations/commit/d6abb83f4a7219788ee9460b347d4f00e2e0a111) 

I think it should be 211, not 210 :)

Reply:
ping @DanMHammer